eclipse-theia/theia
# apply debug icon vscode colors

#### What it does


- fix #6794: apply debug icon vscode colors
- closes #7605: superseded

TODO:
- [ ] file a CQ

#### How to test


- test below for different themes
- test that breakpoints are rendered properly
  - for different kind: normal, inline, conditional, logpoint, functional
  - for different states: enabled, disabled, unverified, unsupported
  - for different views: editor, debug view
  - notice that breakpoints won't necessary look the same way as in latest VS Code, since we use 7 months old Monaco
- test that debug actions are rendered properly in the debug view
- test that logged output in the debug console is rendered properly
- that that debug codicon are themed properly:
  - instal Java Test Runner extension from `Extensions` view
  - check out and open https://github.com/che-samples/console-java-simple
  - open test view
  - check that toolbar and inline actions are colored like in VS Code

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
